include/zephyr/posix/posix_features.h

```diff
@@ -318,10 +318,6 @@
 #define PAGE_SIZE CONFIG_POSIX_PAGE_SIZE
 #endif
 
-#ifndef PAGESIZE
-#define PAGESIZE PAGE_SIZE
-#endif
-
```

Comment:
```suggestion
#define PAGESIZE PAGE_SIZE
```

Reply:
Should the conditions around the definitions of `PAGE_SIZE` and `ATEXIT_MAX` just above also be removed to ensure they are defined correctly?